src/components/BottomModal/BottomModal.js

```diff
@@ -25,26 +39,64 @@ class BottomModal extends React.Component {
       },
     } = e;
 
+    selectedItems.map(value => {
+      console.log(value.name === name ? '중복' : '와우');
+    });
+
     if (isRequired || isSelected) {
       this.setState({ isRequired: false, isSelected: false });
     }
 
     const selectedItemObj = {
       id: selectedItems.length,
       name,
-      price,
+      price: parseInt(price),
       quantity: 1,
     };
     const newSelectedItems = selectedItems.concat(selectedItemObj);
-    this.setState({ selectedItems: newSelectedItems });
+    this.setState({ selectedItems: newSelectedItems }, () => {
+      this.calculateMoney();
+      this.calculateQuantity();
+      this.validateItems();
+    });
   };
 
-  countQuantity = idx => {
+  calculateQuantity = () => {
     const { selectedItems } = this.state;
+    const totalCount = selectedItems.reduce((acc, cur) => {
+      return acc + cur.quantity;
+    }, 0);
+    this.setState({ totalCount });
+  };
```

Comment:
- 예를 들어서, calculateQuantity 함수는 총 갯수를 반환합니다! (setState 하는 것이 아니라)
```suggestion
  calculateQuantity = () => {
    const { selectedItems } = this.state;
    const totalCount = selectedItems.reduce((acc, cur) => {
      return acc + cur.quantity;
    }, 0);
    return totalCount;
  };
```
- 이후에 UI에서 총 갯수가 나타나야 하는 곳에 이 함수를 실행시켜주시면 됩니다!
```jsx
<div className='totalCount'>{this.calculateQuantity()}</div>
```

Reply:
👍